UEFIExtract/ffsdumper.h

```diff
@@ -45,6 +45,7 @@ class FfsDumper
 
 private:
     USTATUS recursiveDump(const UModelIndex & root, const UString & path, const DumpMode dumpMode, const UINT8 sectionType, const UString & guid);
+	void fixFileName(UString &name);
```

Comment:
Something strange with whitespace

Reply:
fixed in commit f71da12ec19f86d842648eeef3e2b4580c4f99d5